tuxemon/resources/db/encounter/mansion.json

```diff
@@ -3,10 +3,10 @@
 
 
 
-{ "monster": "cairfrey","encounter_rate":10,"level_range": [13,16]},
-{ "monster": "polyrock","encounter_rate":10,"level_range": [13,16]},
-{ "monster": "djinnbo","encounter_rate":5,"level_range": [14,17]},
-{ "monster": "pigabyte","encounter_rate":10,"level_range": [13,16]}
+{ "monster": "cairfrey","encounter_rate":6,"level_range": [13,16]},
```

Comment:
As the calculation doesn't underestimate the probabilities any more, the numbers need lowering a bit.